tools/walletextension/container/walletextension_container.go

```diff
@@ -49,18 +54,48 @@ func NewWalletExtensionContainerFromConfig(config config.Config, logger gethlog.
 	userAccountManager := useraccountmanager.NewUserAccountManager(unAuthedClient, logger, databaseStorage, hostRPCBindAddr)
 
 	// add default user (when no UserID is provided in the query parameter - for WE endpoints)
-	userAccountManager.AddAndReturnAccountManager(hex.EncodeToString([]byte(wecommon.DefaultUser)))
+	defaultUserAccountManager := userAccountManager.AddAndReturnAccountManager(hex.EncodeToString([]byte(wecommon.DefaultUser)))
+
+	// add default user to the database (temporary fix before removing wallet extension endpoints)
+	accountPrivateKey, err := crypto.GenerateKey()
+	if err != nil {
+		logger.Error("Unable to generate key pair for default user", log.ErrKey, err)
```

Comment:
for all these error handlings in the constructor, I think the gateway should panic. 
There is no point continuing

Reply:
fixed